### ISSUE TITLE
HBM's Nuclear Tech Mod Compatibility

### ISSUE THREAD

**Issue by bHXDgiuMa5yLHyC8** [GregTech6/gregtech6#11]
The latest pre-release fixed the loading time issue:
https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/releases/tag/1.0.27X3493

Other mods I hope you consider adding compatibility for:
Vic's Modern Warfare Mod ( https://www.curseforge.com/minecraft/mc-mods/vics-modern-warfare-mod )
UpToDateMod ( https://www.curseforge.com/minecraft/mc-mods/uptodatemod )
NetheritePlus+ ( https://www.curseforge.com/minecraft/mc-mods/netheriteplus )
Matter Megadrive ( https://www.curseforge.com/minecraft/mc-mods/matter-megadrive )
LanteaCraft ( https://www.curseforge.com/minecraft/mc-mods/lanteacraft/files/2230557 )
Modular Powersuits ( https://www.curseforge.com/minecraft/mc-mods/modular-powersuits )
MineFantasy II ( https://www.curseforge.com/minecraft/mc-mods/minefantasy2 )

**Comment by GregoriusT** [comment]
NetheritePlus does have a chance to be added, the others I am not sure about.

**Comment by bHXDgiuMa5yLHyC8** [comment]
wiki documents most of the features and is regularly updated:

[https://ntm.fandom.com/wiki/HBM%27s_Nuclear_Tech_Wiki](url)

**Comment by OvermindDL1** [comment]
Okay, so I was looking to this "HBM's Nuclear Tech Mod" to be able to add compat for things like GT6 and HEXCraft, however I am unable to find a maven for it and the git for it at https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT.git (not able to find any other) is not compileable to be able to build a proper dependency for it, so I'm not seeing that this will ever be done until either it gets a proper maven repo or it gets a properly set up git project (look at GT6 or HEXCraft git repo's as example, they are a trivial `git clone ...` and `./gradlew build` in their directories and it all 'just works').

**Comment by OvermindDL1** [comment]
Cross posted from:  https://github.com/CelesTek-Team/HexCraft/issues/22

**Comment by OvermindDL1** [comment]
For note, if help is wanted to be able to get the git repo for the "HBM's Nuclear Tech Mod" feel free to contact me on IRC at espernet, I'm OvermindDL1 there.

**Comment by GregoriusT** [comment]
I just looked through the "API" of HBM's, and I have to say there is absolutely nothing workable there when it comes to Energy Systems (lots of empty marker interfaces), I cant even use the RF API as cheesy bypass, because HBM's happens to use the broken half of the CoFH API, which gets randomly overloaded by wrongly implemented Mods to the point of making the overloaded half of the RF API impossible to use.

IEnergyReceiver doesn't always work, is what I am saying, IEnergyHandler does work though, as long as you make sure that you dont use the super.functions of the extended Interfaces.

**Comment by OvermindDL1** [comment]
Is this done?  Or what's the status?

**Comment by GregoriusT** [comment]
I have no Idea how HBMs works tbh, I only did what I usually do, but no Energy Compat.

**Comment by HbmMods** [comment]
The HE API is still pending, the one for battery items should already work. That'll come with the energy network rework since that still uses godawful 2016 code.

**Comment by GregoriusT** [comment]
Yeah I need to first find out how the Power System even works before I can do Stuff, not to mention what the conversion Ratios are.

**Comment by bHXDgiuMa5yLHyC8** [comment]
Netherlicious ( [https://www.curseforge.com/minecraft/mc-mods/netherlicious](https://www.curseforge.com/minecraft/mc-mods/netherlicious) )

**Comment by GermanGamerMax** [comment]
I think adding vic's mw mod would be awesome, tried running a server with both and all 3d elements of HBM just dissapeared.

**Comment by GregoriusT** [comment]
> I think adding vic's mw mod would be awesome
Uh what Mod?
> tried running a server with both and all 3d elements of HBM just dissapeared.
What exactly happened there? HBM still develops his Mod so it might be worth a report at his place.
